react/src/components/ManageMapProjectPanel/SaveTabContent.tsx

```diff
@@ -110,7 +110,7 @@ const SaveTabContent: React.FC<SaveTabProps> = ({ project }) => {
                   rel="noreferrer"
                   style={{ padding: 0 }}
                 >
-                  {project.system_id}
+                  My Data
```

Comment:
If we show `My Data` here for `designsafe.storage.default`, we should change the `project.system_path` above to be handled similarly to the saveLocationDisplay: https://github.com/TACC-Cloud/hazmapper/blob/33398cc4858a0ccf7f762ba9c9460b68c9312425/react/src/components/CreateMapModal/CreateMapModal.tsx#L99

Reply:
good idea.  gonna merge current PR and draft up this change.